lib/dialects/postgres/index.js

```diff
@@ -40,6 +40,9 @@ PostgresDialect.prototype.supports = _.merge(_.cloneDeep(AbstractDialect.prototy
     using: 2,
     where: true
   },
+  inserts: {
```

Comment:
`inserts` should be removed from here as well, or postgres wont override correct option